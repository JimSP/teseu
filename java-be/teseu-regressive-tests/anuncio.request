POST /imovel/create
accept:application/json
Content-Type:application/json

{

"areaUtil":"90",

"areaTotal":"350",

"complemento":"bloco jdk",

"condominio":400,

"descricao":"testando payloads",

"descricaoPermuta":"",

"flAceitaImovel":"N",

"flCondominioInc":"N",

"flDestaque":"N",

"flFifty":"S",

"flIptuInc":"N",

"flLogradouro":"S",

"idAreasComum":[

1,

2,

4
],

"idCaracteristicas":[

1,

3,

5
],

"idCliente":516,

"idLogradouro":3067080,

"idStatusObra":1,

"idTipoAnuncio":4,

"idTipoImovel":1,

"iptu":200,

"numero":"12",

"numLatitude":"-23.5380005",

"numLongitude":"-46.5723826",

"permutas":[

],

"permutasTipoImovel":[

],

"valorImovel":"222449",

"cadPercent":100,

"nomeBairro":"",

"nomeRua":"",

"qtdBanheiro":"2",

"qtdQuartos":"2",

"qtdSuite":"1",

"qtdVaga":"1"

}