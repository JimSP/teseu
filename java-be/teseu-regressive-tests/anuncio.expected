200=$anuncio.httpStatus
516=$anuncio.responseBody.cliente.id
1=$anuncio.responseBody.cliente.idTipoCliente
Raquel Corretora=$anuncio.responseBody.cliente.nome
Associado=$anuncio.responseBody.cliente.nomeTipoCliente
0=$anuncio.responseBody.cliente.telPrincipalWhats